pkg/cmd/stack/translate.go

```diff
@@ -561,6 +561,9 @@ func translateService(svcName string, s *model.Stack) *apiv1.Service {
 	annotations := translateAnnotations(svc)
 	if s.Services[svcName].Public && annotations[model.OktetoAutoIngressAnnotation] == "" {
```

Comment:
This should work even if `s.Services[svcName].Public` is false, no?

Reply:
We set it to true when there are exposed ports. This way we could support okteto-stack previous implementation